### ISSUE TITLE
Task 6: Testing Data

### ISSUE THREAD

**Issue by tobielf** [tobielf/SER502-Spring2017-Team10#17]
We have formed up the grammar rule for our language, so before we start coding, we can write the test case for our language now. I created eight subdirectories under the data folder. Each member in charge of two cases based on the subdirectories name.

@kkmacart Program1-2,
@XimingFeng Program3-4,
@run-dong-zhu Program5-6,
@tobielf Program7-8.

**Notice!!!**
**Before you start** to create the test cases, comment below to let others know what are you going to write in your test program. The purpose is test cases should not be identical. For example, if I mentioned in the comment, I am going to write a factorial of N, then others can not write the same testing program.

 Here are the detail steps for adding the test case.
1.**Create** a new branch from master named "test-case-YOURNAME"
2.**Modify** the files under each subdirectory assigned to you(see above) following these rules:
- **code.ten**, write the code in our language, try to cover all the constructs in the grammar rule.
- **parsing-tree.png** using [ANTLR](http://www.antlr.org) to generate the parsing tree of the program you wrote in **code.ten**
- **byte-code.asm** based on the parsing tree and the byte-code provided by the runtime environment, write the low-level instructions.
- **README** describe the **purpose** of your testing program, like calculating the factorial of N. and the **expecting output** of your program.

3.**Send** the Pull/Request after you finished your job to let others review your job.
4.**After reviewed** you can merge into the master.

The due date is today(April 15).
